docs/editors/overview.md

```diff
@@ -234,6 +234,8 @@ out-of-the-box.
 **Few steps**: installing Metals requires a few custom steps and minimal
 configuration to work.
 
+Go to the page for your editor to find out how to install.
```

Comment:
```suggestion
_You can find instructions on how to install Metals for you editor on your editor-specific page._
```

What do you think about something like this instead? These sections are just meant to serve as an explanation to the table above, but I suppose it's a good idea to clarify where to get installation instructions like this if it's causing confusion.

Reply:
If it were me I would further even add links to each editor page in the text.

I think you have a small typo: "for _your_ editor".

Reply:
> I think you have a small typo: "for your editor".

Correct, good catch.  As for the link to each editor, I think a note here and then the side bar should be enough to point people in the right direction.